src/sentry/models/release.py

```diff
@@ -261,6 +268,72 @@ def filter_by_stage(
     def order_by_recent(self):
         return self.order_by("-date_added", "-id")
 
+
+class ReleaseModelManager(models.Manager):
```

Comment:
Why is this showing up as new? I think there might be a rebase issue from master, since https://github.com/getsentry/sentry/pull/28722 moved this around. This might be related to some of the test failures.

Reply:
ohh, got it